src/code/z_message.c

```diff
@@ -2241,6 +3008,7 @@ void Message_DrawMain(PlayState* play, Gfx** p) {
                                                  &gSfxDefaultFreqAndVolScale, &gSfxDefaultFreqAndVolScale,
                                                  &gSfxDefaultReverb);
                         }
+                        if (!msgCtx->ocarinaStaff) {} // fake
```

Comment:
```suggestion
                        if (msgCtx->ocarinaStaff == NULL) {} // fake
```
Does this work? We generally avoid using `!` on pointers